Makefile

```
@@ -1,4 +1,5 @@
 HYPERFINE := $(shell command -v hyperfine 2> /dev/null)
+IMG ?= policy-server:local
```

I don't think we need the `docker-build` target, but if we add it, I would change the default tag here. It is not coming from a repo such as ghcr.io already.

```suggestion
IMG ?= policy-server:latest
```